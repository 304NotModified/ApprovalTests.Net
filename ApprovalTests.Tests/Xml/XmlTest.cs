using ApprovalTests.Xml;
using NUnit.Framework;

namespace ApprovalTests.Tests.Xml
{
	[TestFixture]
	public class XmlTest
	{
		[Test]
		public static void TestXml()
		{
			XmlApprovals.VerifyXml("<xml><hello/><start>hi</start></xml>");
		}

	    [Test]
	    public void TestXmlWithAttributes()
	    {
			XmlApprovals.VerifyXml("<xml b=\"123\" a=\"456\"><hello/><start>hi</start></xml>");
	    }
	}
}