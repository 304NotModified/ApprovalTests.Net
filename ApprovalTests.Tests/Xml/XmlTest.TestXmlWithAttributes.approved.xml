<xml a="456" b="123">
  <hello />
  <start>hi</start>
</xml>